src/bookshelf/src/bksw_sql.erl

```diff
@@ -226,7 +226,7 @@ init_transfer_state() ->
     #file_transfer_state{
        size = 0,
        next_chunk = 0,
-       hash_context_md5 = crypto:hash_init(md5),
+       hash_context_md5 = erlang:md5_init(),
```

Comment:
This will generally be required for FIPS irrespective of sig_v4.

Reply:
Sounds good.  It passes all tests, so feel free to merge.